### PR TITLE
Story/add logger

### PR DESCRIPTION
**Pull request by jeffdeville**
Removes the Graylog-related middleware, in favor of the use of a more flexible 'logger' parameter.  There's a new middleware that works the way the Graylog middleware did, except that the data is pushed to the logger that was configured in the parent app.

### CODE REVIEW

**Comment by justincampbell** [comment]
please remove the wip commit

**Comment by jeffdeville** [comment]
When we merge, don't we create a single merge commit, rather than rebase?

**Comment by justincampbell** [comment]
If you merge locally, you could do that. This button will not.                             :point_down: :

**Comment by jeffdeville** [comment]
Ok, commits squashed

**Comment by justincampbell** [comment]
And rubocop

File: lib/stacker_bee/middleware/log_response.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        return unless env.logger
+        params = env.request.params.to_a.sort
+        log_data = {
+          request_path: env.request.path,
+          params: params,
+          response_body: env.raw_response[:body],
+        }
+
+        if env.response.success?
+          log_data[:short_message] = params.find { |key, _| key == "command" }.last
```
Comment (nicholaides) [inline lib/stacker_bee/middleware/log_response.rb]: This is duplicated on line 17

**Pull request merged**